server/protocols/jmap/doc/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:2.7.11
+
+RUN pip install mkdocs
+
+ADD . /doc
```

Comment:
COPY is preferred (see:https://docs.docker.com/engine/articles/dockerfile_best-practices/#add-or-copy)